trunk/as0006321/task_01/src/CMakeLists.txt

```
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 3.10)
+project(Task01)
+
+add_executable(Task01 main.cpp)
```

Кто-то уже использовал название экзешника "Task01" поэтому не собирается. Можете так попробовать заменить, чтоб совпадений не было. 
```suggestion
add_executable(Task01_6321 main.cpp)
```